### ISSUE TITLE
Interpreter PR Todo Items

### ISSUE THREAD

**Issue by fmatthew5876** [EasyRPG/Player#1859]
While we slog through the interpreter PR's here is the list of todo items which are being postponed to avoid rebase issues

- [x] Remove `Game_Map::RefreshType`
- [x] Cleanup `Game_Interpreter::GetFrame()`
- [x] Streamline `Game_Map::RequestMap()` logic in one place. Currently in `Game_Player`, `Scene_Map`, and `Scene_Debug`. Should be after #1784 is merged.
- [x] Remove Async globals and store the async reason and parameters in the `AsyncContext`
- [x] Should we verify save game command chunks against event commands in #1884? How does RPG_RT do it?
- [x] Are level up / skill learned messages word wrapped? Check EXP, Level, and Class Change.
- [x] Can self destruct / escape be undone with show enemy command? - YES
- [ ] Verify what happens if we change event page after event erased. What happens to LSD chunks?

**Comment by Ghabry** [comment]
TBD: Fix performance issues caused by #1740 . Need to confirm my test was correct when back home.

**Comment by fmatthew5876** [comment]
Confirmed the following in RPG_RT:

- [x] Enemy escapes, event show enemy - reappears
- [x] Enemy self-destructs, event show enemy - reappears
- [x] Enemy dies, event show enemy - stays dead

**Comment by fmatthew5876** [comment]
Remove the MIDI bullet as thats already tracked by #1733
